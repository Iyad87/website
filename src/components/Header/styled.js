import styled from 'styled-components';
import { gridColWidth } from '../../styles/spacing';

export const Wrapper = styled.header`
  background-color: white;
  display: flex;
  justify-content: space-between;
  padding: 30px ${gridColWidth}px;
  align-items: center;
`;
