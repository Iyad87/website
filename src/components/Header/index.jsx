import React from 'react';

import Logo from '../Logo';
import Navigation from '../Navigation';
import { Wrapper } from './styled';

export default function Header() {
  return (
    <Wrapper>
      <Logo />
      <Navigation />
    </Wrapper>
  );
}
