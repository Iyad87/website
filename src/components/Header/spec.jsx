import React from 'react';
import { shallow } from 'enzyme';
import Header from './index';

test('components/Header snapshot', () => {
  const wrapper = shallow(<Header siteTitle="YLD" />);

  expect(wrapper).toMatchSnapshot();
});
