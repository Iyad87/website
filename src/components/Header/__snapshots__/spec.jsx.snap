// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Header snapshot 1`] = `
<styled__Wrapper>
  <Logo />
  <Navigation />
</styled__Wrapper>
`;
